src/Sarif.ValidationTests/SarifValidatorTests.cs

```diff
@@ -36,6 +36,8 @@ public void ValidatesAllTestFiles()
             foreach (string inputFile in TestCases)
             {
                 string instanceText = File.ReadAllText(inputFile);
+                instanceText = Utilities.UpdateVersionNumberToCurrent(instanceText);
```

Comment:
>UpdateVersionNumberToCurrent [](start = 41, length = 28)

we're going to need a general utility that upgrades in-flight v2 SARIF. ideally, users will be able to seamlessly load their older SARIF v2 which will be automatically transformed to the current version. if we don't build this, we'll need to provide API snapshots for all versions and require users to handle their own fix-ups. not a good idea. let's huddle with scott louvau on an efficient approach for this, he'll have good ideas to offer.